e2e/dashboardSpec.js

```diff
@@ -1,19 +1,16 @@
-/*global describe,it,expect,$$,element,browser,by*/
-
 var testMethod = process.env.TRAVIS ? xdescribe : describe;
 
-testMethod('Dashboard', function () {
+describe('Dashboard', function () {
     'use strict';
 
     beforeEach(function () {
         browser.get(browser.baseUrl);
-
-        browser.driver.wait(function () {
-            return browser.driver.isElementPresent(by.css('.panel:nth-child(1)'));
-        }, 5000); // wait 5s
     });
 
     it('should display a navigation menu linking to all entities', function () {
+        browser.driver.wait(function () {
```

Comment:
It can, but in test, I prefer not factorize too quickly and being as mush as explicit as possible.
IMO, code duplicatation is far better than taking few hours to debug a test.